test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1

```diff
@@ -0,0 +1,111 @@
+﻿Describe "ConvertTo-Html DRT Unit Tests" -Tags "CI" {
+    $customObject = [pscustomobject]@{"Name" = "John Doe"; "Age" = 42; "Friends" = ("Jack", "Jill")}
+    $newLine = [System.Environment]::NewLine
```

Comment:
Done. New commit pushed up.

From: Mike Richmond [mailto:notifications@github.com]
Sent: Tuesday, September 13, 2016 1:51 PM
To: PowerShell/PowerShell <PowerShell@noreply.github.com>
Cc: Jeff Bienstadt <JeffBi@jetstreamsoftware.com>; Mention <mention@noreply.github.com>
Subject: Re: [PowerShell/PowerShell] Make ConvertTo-Html work on CoreCLR (#2241)


In test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1<https://github.com/PowerShell/PowerShell/pull/2241#discussion_r78641559>:

> @@ -0,0 +1,111 @@

> +﻿Describe "ConvertTo-Html DRT Unit Tests" -Tags "CI" {

> +    $customObject = [pscustomobject]@{"Name" = "John Doe"; "Age" = 42; "Friends" = ("Jack", "Jill")}

> +    $newLine = [System.Environment]::NewLine

This test code should go within BeforeAll instead of existing within the Describe. See Pester Do's and Don'ts<https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/PesterDoAndDont.md>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/PowerShell/PowerShell/pull/2241/files/e8481e6a53c1d6d2e84c09dc22c2fa94fa8dcf67#r78641559>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AGZehET8cnXNuO1cSEqZ2TNnZ-g7Ffa6ks5qpwy7gaJpZM4J7Etq>.